.github/workflows/test.yml

```
@@ -0,0 +1,26 @@
+name: Cargo test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

```suggestion
      - main
      - develop
```

`master` doesn’t exist in this repository.